bin/pre-commit

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+files=$(git diff --cached --name-only --diff-filter=ACM | grep ".jsx*$")
```

Comment:
this git expression should be `$(git diff --cached --name-only src/ test/ | grep ".jsx*$")` instead and substituting it in line 12. In [Calypso](https://github.com/Automattic/wp-calypso/blob/master/bin/pre-commit) this check was some legacy thing that was removed recently.